documentation/manual/src/main/asciidoc/modules/configuration.asciidoc

```diff
@@ -577,3 +629,7 @@ Download the Infinispan modules pack for WildFly {wildfly-short-version} from he
 
 Then similarly to what you did with the Hibernate OGM modules zip, unpack this one too in your `modules` directory within the application server.
 
+[NOTE]
+====
+This distribution is already provided by Hibernate OGM Infinispan Remote and Embedded feature packs.
```

Comment:
I think it's hard to understand what you mean with `This distribution`.

Is this paragraph still relevant? It refers to Hibernate OGM modules and they don't exist anymore.

I would change the note to something like:
```
If you are using the Hibernate OGM  Infinispan feature packs, you don't have to worry about this. Infinispan ... is already 
inlcuded in them.
```

Reply:
No the paragraph is not relevant anymore. Feature pack solving the problem of provision of Infinispan module too. I'll remove it.